packages/salesforcedx-vscode-core/package.json

```diff
@@ -27,7 +27,7 @@
   "dependencies": {
     "@heroku/functions-core": "0.7.0",
     "@octokit/plugin-paginate-rest": "2.21.3",
-    "@salesforce/core": "^3.31.18",
+    "@salesforce/core": "^3.33.6",
```

Comment:
Elsewhere this is set at 3.31.18 - does this need to be consistent?

Reply:
I think as long as the utils-vscode and vscode-core packages get the same version of core installed everything is ok.  But not sure if we should align the version in the other pkgs as part of this PR.